.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -448,6 +448,20 @@ jobs:
         run: |
           # Ensure the working directory gets chowned back to the current user
           docker run --rm -v "$(pwd)":/v -w /v "${ALPINE_IMAGE}" chown -R "$(id -u):$(id -g)" .
+      - uses: driazati/upload-artifact-s3@21c31d0a7bcb056ca50bd6ce197ba6507c26a1be
```

Comment:
the only change of note in this fork is [to use a different prefix based on the PR number rather than the workflow run ID](https://github.com/seemethere/upload-artifact-s3/compare/main...driazati:main#diff-519fa009c5415a88d003e807b2f95d82d6acdd9651c8352aa7f6f9f808c44a0fR39)